src/app/search/FilterHelp.tsx

```diff
@@ -0,0 +1,700 @@
+                <span>is:stackable</span>
+              </td>
+              <td>{t('Filter.Stackable')}</td>
+            </tr>
+            <tr>
+              <td>
+                <span>is:autorifle</span> <span>is:fusionrifle</span>
+                {destinyVersion === 2 && <span>is:grenadelauncher</span>}
+                <span>is:handcannon</span> <span>is:machinegun</span> <span>is:pulserifle</span>
+                <span>is:scoutrifle</span> <span>is:shotgun</span> <span>is:sidearm</span>
+                <span>is:sniperrifle</span> {destinyVersion === 2 && <span>is:submachine</span>}
+                {destinyVersion === 2 && <span>is:bow</span>} <span>is:rocketlauncher</span>
+                <span>is:sword</span>
+              </td>
+              <td>{t('Filter.WeaponType')}</td>
+            </tr>
+            <tr>
+              <td>
+                <span>is:engram</span>
+              </td>
+              <td>{t('Filter.Engrams')}</td>
+            </tr>
+
+            {destinyVersion === 1 && (
+              <tr>
+                <td>
+                  <span>is:glimmeritem</span> <span>is:glimmerboost</span>{' '}
+                  <span>is:glimmersupply</span>
+                </td>
+                <td ng-i18next="Filter.Glimmer.Glimmer">
+                  <ul>
+                    <li>{t('Filter.Glimmer.Any')}</li>
+                    <li>{t('Filter.Glimmer.Boost')}</li>
+                    <li>{t('Filter.Glimmer.Item')}</li>
+                  </ul>
+                </td>
+              </tr>
+            )}
+            <tr>
+              <td>
+                <span>is:new</span>
+              </td>
+              <td>{t('Filter.NewItems')}</td>
+            </tr>
+            <tr>
+              <td>
+                <span>tag:none</span> <span>tag:favorite</span> <span>tag:keep</span>
+                <span>tag:junk</span> <span>tag:infuse</span>
+              </td>
+              <td>
+                <ul>
+                  <li>{t('Filter.Tags.NoTag')}</li>
+                  <li>{t('Filter.Tags.Favorite')}</li>
+                  <li>{t('Filter.Tags.Keep')}</li>
+                  <li>{t('Filter.Tags.Dismantle')}</li>
+                  <li>{t('Filter.Tags.Infuse')}</li>
+                </ul>
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <span>notes:value</span>
+              </td>
+              <td>{t('Filter.Notes')}</td>
+            </tr>
+
+            {$featureFlags.reviewsEnabled && (
+              <>
+                <tr>
+                  <td>
+                    <span>is:rated</span>
+                  </td>
+                  <td>{t('Filter.Rated')}</td>
+                </tr>
+                <tr>
+                  <td>
+                    <span>rating:value</span> <span>rating:&gt;=value</span>{' '}
+                    <span>rating:&gt;value</span>
+                    <span>rating:&lt;value</span> <span>rating:&lt;=value</span>
```

Comment:
```
                    <span>rating:&gt;value</span> <span>rating:&lt;value</span>
                    <span>rating:&lt;=value</span>
```

Reply:
Fixed all these, but they don't matter since we style them to display in a column.

Reply:
I just figured prettier would of handled it.